### PR TITLE
Change `decl_storage!` to import the storage traits automatically

### PR DESCRIPTION
**Pull request by expenses**
See #3672.

### CODE REVIEW

File: srml/system/src/lib.rs [hunk 1]
```diff
@@ -111,7 +111,7 @@ use sr_primitives::{
 
 use primitives::storage::well_known_keys;
 use support::{
-	storage, decl_module, decl_event, decl_storage, StorageDoubleMap, StorageValue, StorageMap,
+	storage, decl_module, decl_event, decl_storage,
```
Comment (kianenigma) [inline srml/system/src/lib.rs]:
```suggestion
	decl_module, decl_event, decl_storage, decl_error, storage, Parameter,
	traits::{Contains, Get},
```

File: srml/system/src/lib.rs [hunk 2]
```diff
@@ -111,7 +111,7 @@ use sr_primitives::{
...
 	Parameter, traits::{Contains, Get}, decl_error,
```
Comment (kianenigma) [inline srml/system/src/lib.rs]:
```suggestion

```

File: srml/im-online/src/lib.rs [hunk 3]
```diff
@@ -86,8 +86,7 @@ use sr_staking_primitives::{
 	offence::{ReportOffence, Offence, Kind},
 };
 use support::{
-	decl_module, decl_event, decl_storage, print, ensure,
-	Parameter, StorageValue, StorageDoubleMap,
+	decl_module, decl_event, decl_storage, print, ensure, Parameter
```
Comment (kianenigma) [inline srml/im-online/src/lib.rs]: probably fits in 1 line

File: node/runtime/src/lib.rs [hunk 4]
```diff
@@ -85,7 +85,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 162,
-	impl_version: 162,
+	impl_version: 163,
```
Comment (bkchr) [inline node/runtime/src/lib.rs]:
```suggestion
	impl_version: 162,
```

**Pull request merged**